### PR TITLE
Alpha2 release notes

### PR DESCRIPTION
**Pull request by akeller**
## Description

All known epics & features have been added, including internal links (public or private, as comments). 

Note - some links to new docs generated for new epics are not live yet. 

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [x] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

**Comment by akeller** [comment]
@camunda/tech-writers, I've marked this as ready for review, but have not done any editing myself. We do not yet have a standard format or "style" for release notes, but we can discuss this as a team.

The top two epics/features on the PR were content I generated from the descriptions, value prop, and implementation tickets. We'll need @sbuettner or a delegate to review those.

**Review by sbuettner — COMMENTED** [review]
Great @akeller what do you think about grouping and/or ordering release notes by component?

File: docs/reference/release-notes/860.md [hunk 1]
```diff
@@ -12,6 +17,78 @@ keywords:
 | ---------------------- | ---------------------------- | ------------ | ------------ |
 | 8 October 2024         | 14 April 2026                |              |              |
 
+## 8.6.0-alpha2
+
+| Release date | Changelog(s)                                                                                                                                                              |                   |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
+| 11 June 2024 | - [ Camunda 8 core ](https://github.com/camunda/zeebe/releases/tag/8.6.0-alpha2) <br /> - [ Connectors ](https://github.com/camunda/connectors/releases/tag/8.6.0-alpha2) | [Release blog](#) |
```
Comment (sbuettner) [inline docs/reference/release-notes/860.md:24]:
```suggestion
| 11 June 2024 | - [ Camunda 8 core ](https://github.com/camunda/zeebe/releases/tag/8.6.0-alpha2) <br /> - [ Connectors ](https://github.com/camunda/connectors/releases/tag/8.6.0-alpha2.1) | [Release blog](#) |
```

**Comment by akeller** [comment]
> Great @akeller what do you think about grouping and/or ordering release notes by component?

@sbuettner I'll consider this in another iteration. For now, I want to focus on surfacing the right items in the release notes. Since alphas are generally smaller, I'm hoping to get to this for the next minor.

File: docs/reference/release-notes/860.md (hunk 1 above)
Comment (akeller) [reply docs/reference/release-notes/860.md:24]: For future reference, do you usually release `alphax.0` or `alphax.1`? Or will you keep track of this with the Connectors team?

Comment (sbuettner) [reply docs/reference/release-notes/860.md:24]: We usually only release using the `alpha-X` pattern without any patches like everyone else. In this case we had to release a patch for the `alpha2` hence `alpha2.1` as we did not want to go with `alpha3`.

**Comment by sbuettner** [comment]
Alright, I thought it would be nice to see them grouped together somehow so people can navigate the release notes more easily. Thanks for your support!

**Review by christinaausley — APPROVED** [review]

**Pull request merged**